storage/block/reference-content/advantages-migrating-to-sbs.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: Advantages of migrating your Block Storage legacy volumes and snapshots to Block Storage Low Latency
+  description: Learn the key advantages of migrating your Block Storage legacy volumes and snapshots to Block Storage Low Latency.
+content:
+  h1: Advantages of migrating your Block Storage legacy volumes and snapshots to Block Storage Low Latency
+  paragraph: Learn the key advantages of migrating your Block Storage legacy volumes and snapshots to Block Storage Low Latency.
+tags: volumes block-storage 5kiops 10kiops sbs low-latency
+dates:
+  validation: 2025-01-03
+  posted: 2025-01-03
+categories:
+  - storage
+---
+
+## Transitioning from the Instance API to the Block Storage API for volume and snapshot management
+
+Scaleway is transitioning the management of all Block Storage legacy volumes and snapshots from the [Instance API](https://www.scaleway.com/en/developers/api/instance/) to the [Block Storage API](https://www.scaleway.com/en/developers/api/block/). All Block Storage volumes and snapshots created via the [Scaleway console](https://console.scaleway.com) will no longer be available from the [Compute section](https://console.scaleway.com/instance/), but from the [Storage section](https://console.scaleway.com/block-storage).
+
+This change provides you with enhanced performance and advanced storage capabilities. If you have created volumes and snapshots via the Instance API, we **strongly recommend migrating them to the Block Storage API** to benefit from these improvements.
+
+Local volumes will remain exclusively supported by the Instance API.
+
+## Key advantages of using the Block Storage API
+
+By migrating your Block Storage volumes and snapshots to the Block Storage API, you take advantage of the following benefits:
+
+- **Low latency**: volumes managed by the Block Storage API leverage modern hardware, ensuring faster access times and smoother performance.
```

Comment:
Instance API handles bssd volumes which will still be bssd volumes behind the SBS/block API